internal/core/data/interfaces/mocks/DBClient.go

```diff
@@ -220,16 +208,39 @@ func (_m *DBClient) Events() ([]models.Event, error) {
 	return r0, r1
 }
 
+// EventsByChecksum provides a mock function with given fields: checksum
+func (_m *DBClient) EventsByChecksum(checksum string) ([]go_mod_core_contractsmodels.Event, error) {
+	ret := _m.Called(checksum)
+
+	var r0 []go_mod_core_contractsmodels.Event
+	if rf, ok := ret.Get(0).(func(string) []go_mod_core_contractsmodels.Event); ok {
+		r0 = rf(checksum)
+	} else {
+		if ret.Get(0) != nil {
+			r0 = ret.Get(0).([]go_mod_core_contractsmodels.Event)
+		}
+	}
+
+	var r1 error
```

Comment:
can we use standard name like "err".

Reply:
This mock is generated code via `mockery`. Would prefer to not change things like this.